docker-build/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Dockerfile for bitnami/fluentd sidecar image with all the necessary plugins for our log analytic providers
+FROM bitnami/fluentd:latest
+LABEL maintainer="Partner Engineering <partner_support@jfrog.com>"
+
+## Build time Arguments, short circuit them to ENV Variables so they are available at run time also
+ARG SOURCE=JFRT
+ARG TARGET=NEWRELIC
+
+## Environment Variables set by this docker file, there will be seperate env params set by a env file while running the containers
+## For better maintainability always depend dockerfile code on the environment variables declared in this file to add more platforms
+ENV SRC_PLATFORM=$SOURCE
+ENV TGT_PLATFORM=$TARGET
+
+USER root
+
+## Install JFrog Plugins
+RUN fluent-gem install fluent-plugin-jfrog-siem
+RUN fluent-gem install fluent-plugin-jfrog-metrics
+RUN fluent-gem install fluent-plugin-jfrog-send-metrics
```

Comment:
Do these 3 gem installs must be done separately? Can they be installed in single `RUN`? I'm thinking about Docker layer caching, etc.

Reply:
They can be installed in a single line provided the RubyGems version is 2.0 and above, and sometimes the runtime dependencies of all the plugins in a single command are not installed properly. One of the reasons why when installing a list of gems, they're usually placed in a file with all separately listed and installed with bundler